src/ui/public/filter_bar/filter_view/base_filter_view.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EuiBadge } from '@elastic/eui';
+import { Filter, isFilterPinned } from '@kbn/es-query';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { SFC } from 'react';
+
+interface Props {
+  filter: Filter;
+  title: string;
+  [propName: string]: any;
+}
+
+export const BaseFilterView: SFC<Props> = ({ filter, title, children, ...rest }: Props) => {
+  let badgeTitle = `Filter: ${
+    filter.meta.negate
+      ? ` ${i18n.translate('common.ui.filterBar.negatedFilterPrefix', {
+          defaultMessage: 'NOT ',
+        })}`
+      : ''
+  }${title}. Select for more filter actions.`;
```

Comment:
Don't you need to put "Select for more filter actions" inside a `i18n.translate`?